platform/docs/docs/configuration/url.md

```diff
@@ -110,17 +110,31 @@ You can open more than one study in the Viewer by adding the `StudyInstanceUIDs`
 :::tip
 
 You can ues this feature to open a current and prior study in the Viewer.
-Read more in the [Hanging Protocol Module](../platform/extensions/modules/hpModule.md#matching-on-prior-study-with-uid) section
+Read more in the [Hanging Protocol Module](../platform/extensions/modules/hpModule.md#matching-on-prior-study-with-uid) section.  You can also use commas to separate
+values.
```

Comment:
can you provide couple of examples pleaes? 
- if you want two Studies ...
- if you want to do one study and two series

Do we have filtering for SOP too?
If so, that as well